packages/manager/modules/pci/src/projects/project/project.service.js

```diff
@@ -70,13 +70,15 @@ export default class {
                   if (label === FRAUD_REFUSED)
                     status.isRefusedByAntiFraud = true;
                 });
-
-                if (
-                  status.isManuallyReviewedByAntiFraud ||
-                  status.isRefusedByAntiFraud
-                ) {
-                  status.isActivating = false;
-                }
+              }
+              if (status.isRefusedByAntiFraud) {
+                status.isManuallyReviewedByAntiFraud = false;
+              }
+              if (
+                status.isRefusedByAntiFraud ||
+                status.isManuallyReviewedByAntiFraud
+              ) {
+                status.isActivating = false;
```

Comment:
**question:** Can you explain what you're trying to do here?

Reply:
Sure, the status object we build in this method reflects the steps a pci order goes through. Each property is a boolean which is true is the associated step exists in the order. But some properties should be cleared in case some others are true. For instance, a project should not be flagged as "activating" is the anti fraud team has detected some anomalies in the order

Reply:
Yes I think that is more readable. Can you make the change as you suggested above?